### PR TITLE
ANC Logic - Hepatitis B diagnosis

### PR DESCRIPTION
**Pull request by anvmn**
#137

### CODE REVIEW

**Comment by anvmn** [comment]
@mariano-dagostino please review

There're several `@todo` comments - which are indicators for future development (medication prescription that will be required for other diagnoses).

File: client/src/elm/Backend/Update.elm [hunk 1]
```diff
@@ -3912,45 +3930,101 @@ generatePrenatalAssessmentMsgs currentDate language isChw updateAssesment before
...
+                                signs =
+                                    List.map (Translate.PrenatalDiagnosis >> translate language) urgentDiagnoses
+                                        |> String.join ", "
+
+                                message =
+                                    if isChw then
+                                        translate language Translate.DangerSignsLabelForChw ++ ": " ++ signs
+
+                                    else
+                                        translate language Translate.DangerSignsLabelForNurse ++ " " ++ signs
```
Comment (mariano-dagostino) [inline client/src/elm/Backend/Update.elm:3971]: Any reason why one has `: ` and the other only spaces?

**Review by mariano-dagostino — APPROVED** [review]
@anvmn Looks good. Just a small nitpick

File: client/src/elm/Backend/Update.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Backend/Update.elm:3971]: @mariano-dagostino 

This is per requirements. The labels are different. 
The one with `:` just says `Danger Signs`, while the other says `Patient shows signs of`
```
        DangerSignsLabelForChw ->
            { english = "Danger Signs"
            , kinyarwanda = Just "Ibimenyetso Mpuruza"
            }

        DangerSignsLabelForNurse ->
            { english = "Patient shows signs of"
            , kinyarwanda = Nothing
            }
```

**Pull request merged**